src/libraries/Microsoft.Extensions.Hosting/src/Internal/HostingLoggerExtensions.cs

```diff
@@ -101,5 +101,16 @@ public static void BackgroundServiceStoppingHost(this ILogger logger, Exception?
                     message: SR.BackgroundServiceExceptionStoppedHost);
             }
         }
+
+        public static void HostedServiceStartupFaulted(this ILogger logger, Exception? ex)
+        {
+            if (logger.IsEnabled(LogLevel.Error))
+            {
+                logger.LogError(
+                   eventId: LoggerEventIds.HostedServiceStartupFaulted,
+                   exception: ex,
+                   message: "Hosting failed to start");
```

Comment:
The message should be added to the resources.

See the usage above `message: SR.BackgroundServiceExceptionStoppedHost)`